test/ASTGen/decls.swift

```diff
@@ -31,6 +34,10 @@ import protocol Swift.Sequence
 import func Swift.max
 import var Swift._playgroundPrintHook
 
+
+using @MainActor
+// FIXME: cannot add `using nonisolated` here because it's a re-declaration
```

Comment:
Hah, exactly what I was worried about when I saw the proposal update :) 
Good to see we'll want to diagnose this.